tests/timeseries/_formula_engine/test_formula_composition.py

```diff
@@ -0,0 +1,180 @@
+# License: MIT
+# Copyright © 2022 Frequenz Energy-as-a-Service GmbH
```

Comment:
General suggestion: New files should set 2023 as a year

Reply:
done.